teamd/teamd_lw_ttdp.h

```diff
@@ -404,6 +412,11 @@ struct ab {
 	uint32_t latest_line_slow_timeout_ms;
 	void*(*line_timeout_value_update_func)(void*,void*);
 
+	/* The values in this struct are exposed as state variables, per line, and
+	 * holds counters for how many HELLO frames have been sent/received and
+	 * how many times fast mode has been activated. */
+	struct hello_stats lines_hello_stats[4];
```

Comment:
Maybe use a constant instead of magic number `4`

Reply:
Agreed, fixed!